src/metabase/driver.clj

```diff
@@ -368,18 +368,19 @@
 
 ;; TODO - this is only used (or implemented for that matter) by SQL drivers. This should probably be moved into the
 ;; `:sql` driver. Don't bother to implement this for non-SQL drivers.
-(defmulti date-interval
-  "Return an driver-appropriate representation of a moment relative to the current moment in time. By default, this
+(defmulti date-add
+  "Return an driver-appropriate representation of a moment relative to the given time (or now). By default, this
   returns an `Timestamp` by calling `metabase.util.date/relative-date`; but when possible drivers should return a
   native form so we can be sure the correct timezone is applied. For example, SQL drivers should return a HoneySQL
   form to call the appropriate SQL fns:
 
     (date-interval :postgres :month 1) -> (hsql/call :+ :%now (hsql/raw \"INTERVAL '1 month'\"))"
```

Comment:
Could you update the rest of the docstring too? 🙏

Reply:
I've went and removed the awkward default of returning `du/relative-date` as it didn't make much sense with the new semantics and is something that was used in scaffolding for one test and that's it. That also massively simplifies the docstring.